### ISSUE TITLE
1.20.4 Crash

### ISSUE THREAD

**Issue by davidlaym** [hashalite/Freecam#125]
Getting the following crash report, triggered by just enabling freecam:

```
---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 2023-12-13 19:19:27
Description: Ticking entity

java.lang.NoClassDefFoundError: net/minecraft/class_4262
	at net.xolt.freecam.config.CollisionWhitelist.<clinit>(CollisionWhitelist.java:10)
	at net.minecraft.class_4970$class_4971.handler$zmh000$freecam$onGetCollisionShape(class_4970.java:1445)
	at net.minecraft.class_4970$class_4971.method_26194(class_4970.java)
	at net.minecraft.class_5329.computeNext(class_5329.java:101)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:145)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:140)
	at net.minecraft.class_1941.method_8587(class_1941.java:51)
	at net.minecraft.class_1657.method_52558(class_1657.java:459)
	at net.minecraft.class_1657.method_7318(class_1657.java:422)
	at net.minecraft.class_1657.method_5773(class_1657.java:332)
	at net.minecraft.class_742.method_5773(class_742.java:59)
	at net.minecraft.class_746.method_5773(class_746.java:218)
	at net.minecraft.class_638.method_18646(class_638.java:277)
	at net.minecraft.class_1937.method_18472(class_1937.java:492)
	at net.minecraft.class_638.method_32124(class_638.java:260)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:256)
	at net.minecraft.class_310.method_1574(class_310.java:2011)
	at net.minecraft.class_310.method_1523(class_310.java:1289)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_4262
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 24 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.xolt.freecam.config.CollisionWhitelist.<clinit>(CollisionWhitelist.java:10)
	at net.minecraft.class_4970$class_4971.handler$zmh000$freecam$onGetCollisionShape(class_4970.java:1445)
	at net.minecraft.class_4970$class_4971.method_26194(class_4970.java)
	at net.minecraft.class_5329.computeNext(class_5329.java:101)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:145)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:140)
	at net.minecraft.class_1941.method_8587(class_1941.java:51)
	at net.minecraft.class_1657.method_52558(class_1657.java:459)
	at net.minecraft.class_1657.method_7318(class_1657.java:422)
	at net.minecraft.class_1657.method_5773(class_1657.java:332)
	at net.minecraft.class_742.method_5773(class_742.java:59)
	at net.minecraft.class_746.method_5773(class_746.java:218)
	at net.minecraft.class_638.method_18646(class_638.java:277)
	at net.minecraft.class_1937.method_18472(class_1937.java:492)
	at net.minecraft.class_638.method_32124(class_638.java:260)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:256)

-- Entity being ticked --
Details:
	Entity Type: minecraft:player (net.xolt.freecam.util.FreeCamera)
	Entity ID: -420
	Entity Name: FreeCamera
	Entity's Exact location: 380.55, 116.22, 597.29
	Entity's Block location: World: (380,116,597), Section: (at 12,4,5 in 23,7,37; chunk contains blocks 368,-64,592 to 383,319,607), Region: (0,1; contains chunks 0,32 to 31,63, blocks 0,-64,512 to 511,319,1023)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at net.minecraft.class_1937.method_18472(class_1937.java:492)
	at net.minecraft.class_638.method_32124(class_638.java:260)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_638.method_18116(class_638.java:256)
	at net.minecraft.class_310.method_1574(class_310.java:2011)
	at net.minecraft.class_310.method_1523(class_310.java:1289)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Uptime --
Details:
	JVM uptime: 26.174s
	Wall uptime: 15.648s
	High-res time: 13.893s
	Client ticks: 212 ticks / 10.600s
Stacktrace:
	at net.minecraft.class_310.method_1587(class_310.java:2564)
	at net.minecraft.class_310.method_54580(class_310.java:995)
	at net.minecraft.class_310.method_1514(class_310.java:908)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 2 total; [class_746['davidlaym'/604, l='ClientLevel', x=380.55, y=115.00, z=597.29], FreeCamera['FreeCamera'/-420, l='ClientLevel', x=380.55, y=116.22, z=597.29]]
	Chunk stats: 1024, 210 F: 1129 L: 0 U: 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,106,0), Section: (at 0,10,0 in 0,6,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 183787 game time, 226768 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
	Tracked entity count: 135
```

**Comment by bearaujus** [comment]
got the same issue :c

**Comment by tticehurst** [comment]
Added a PR for a fabric 1.20.4 update 
https://github.com/hashalite/Freecam/pull/127

**Comment by MattSturgeon** [comment]
Here's an unofficial build you can use while we wait for an actual release:

- Fabric: [freecam-fabric-1.2.1.1-1.20.4.zip](https://github.com/hashalite/Freecam/files/13693317/freecam-fabric-1.2.1.1-1.20.4.zip)
- Forge: [freecam-forge-1.2.1.1-1.20.4.zip](https://github.com/hashalite/Freecam/files/13693320/freecam-forge-1.2.1.1-1.20.4.zip)

The jar is inside the zip file (so extract that). Goes in `mods` folder like normal.

**Comment by Zorklis** [comment]
> Here's an unofficial build you can use while we wait for an actual release:
> 
>     * Fabric: [freecam-fabric-1.2.1.1-1.20.4.zip](https://github.com/hashalite/Freecam/files/13693317/freecam-fabric-1.2.1.1-1.20.4.zip)

Works, no crash :)
props to @tticehurst

**Comment by hashalite** [comment]
1.2.2 has been released with 1.20.4 support.
